sumy/utils.py

```diff
@@ -33,8 +33,8 @@ def normalize_language(language):
     return language
 
 
-def fetch_url(url):
-    with closing(requests.get(url, headers=_HTTP_HEADERS)) as response:
+def fetch_url(url, timeout=None):
```

Comment:
It is a good practice to set connection and read timeouts differently. Also, 30 seconds as a default is a generous value. Better than hanging forever. Can you change it please?

```suggestion
def fetch_url(url, timeout=(3.05, 30)):
```

Reply:
Nice. Didn't know I can commit my own suggestions 😄